### PR TITLE
start of updating assimp usage in tutorial 4.1

### PR DESCRIPTION
**Pull request by Artromskiy**
# Summary of the PR
A short summary of this PR and what it adds/removes/fixes.
Work started
# Related issues, Discord discussions, or proposals
Model loading code of tutorial 4.1 copies vertices and uv data float by float instead of assignment of full vector
Links go here.

# Further Comments

### CODE REVIEW

**Comment by Artromskiy** [comment]
@HurricanKai 
Waiting for approval

**Review by ThomasMiz — APPROVED** [review]
All looks good!

**Pull request merged**
